core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -177,12 +177,16 @@ <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplor
     font-family: Roboto;
     font-size: x-small;
     font-weight: bold;
-    left: 18px;
+    left: 14px;
```

Comment:
Please alphabetize CSS attributes within a rule -- it makes it easier to scan for their values, and to avoid duplicates.